anesthetic/plot.py

```diff
@@ -969,7 +969,7 @@ def hist_plot_1d(ax, data, *args, **kwargs):
     xmax = quantile(data, q[-1], weights)
     range = kwargs.pop('range', (xmin, xmax))
 
-    if type(bins) == str and bins in ['knuth', 'freedman', 'blocks']:
+    if type(bins) is str and bins in ['knuth', 'freedman', 'blocks']:
```

Comment:
I believe the more canonical way is actually to use `isintance(bins, str)`.

Reply:
agreed. Updated in c6d173e